CMakeLists.txt

```diff
@@ -41,18 +41,18 @@ include(environmentchecks)
 
 # COMPILER_CACHE  ==> LIBCELLML_COMPILER_CACHE
 set(_PARAM_ANNOTATION "Enable compiler cache.")
-if((WIN32 AND CLCACHE_AVAILABLE) OR (NOT WIN32 AND CCACHE_AVAILABLE))
+if(BUILDCACHE_AVAILABLE OR (WIN32 AND CLCACHE_AVAILABLE) OR (NOT WIN32 AND CCACHE_AVAILABLE))
```

Comment:
This should be reduced to `HAVE_COMPILER_CACHE` in environmentchecks.cmake.

Reply:
The condition also shouldn't rely on `WIN32` for clcache it should be `MSVC`.

Reply:
> This should be reduced to `HAVE_COMPILER_CACHE` in environmentchecks.cmake.

Agreed, not least since this is what I did in libOpenCOR (also it's called `COMPILER_CACHING_AVAILABLE`).

> The condition also shouldn't rely on `WIN32` for clcache it should be `MSVC`.

Agreed.

All done in commit 6a452f55.